src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -257,17 +245,46 @@ export function GET(context) {
 }
 ```
 
-To access the value of the `MY_VARIABLE` binding add the following to your code:
+See the [list of all supported bindings](https://developers.cloudflare.com/workers/wrangler/api/#supported-bindings) in the Cloudflare documentation.
+
+
+### Environment variables and secrets
```

Comment:
Can we specify somewhere that env vars/secrets can be used with `astro:env` too?